packages/electron/src/telemetry/electron-telemetry.ts

```diff
@@ -95,6 +95,16 @@ export const logReactMetrics = (response: any) => {
   reactLogger.info(JSON.stringify(errorMetric));
 };
 
+export const logReactError = (response: any) => {
+  const cur = new Date();
+  let curTimeArr = cur.toISOString().split('T');
+  let curDate = curTimeArr[0];
+  let curTime = curTimeArr[1].split('.')[0];
+  let appVersion = app.getVersion();
+  let errorMessage = `[${curDate} ${curTime} ERR] (${appVersion}) ${response}`
```

Comment:
Use this format 
https://github.com/aws/porting-assistant-dotnet-ui/blob/33c64aaf78d89612bb0d1c443c2a40659a17a81d/packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs#L34

Reply:
Yes, I updated codes. Thanks!